translate/canon/README.md

```diff
@@ -28,6 +28,12 @@ A collection of samples that demonstrate how to call the
     Test execution time: 2.6078 Seconds    
 	```
 
+## Designed to demonstrate Google APIs
+
+These code samples are designed to demonstrate simple code that calls Google
+APIs.  So sometimes, they don't conform to engineering best practices like
+"don't copy and paste code."
```

Comment:
Maybe worth also mentioning that they are optimized for being embedded at cloud.google.com documentation and therefore there is a separate file per snippet.

Reply:
Done.

Thanks for the review!